### PR TITLE
feat: update the `color` prop for `Amount`

### PR DESCRIPTION
**Pull request by snitin315**
To be merged after https://github.com/razorpay/blade/pull/1980

### CODE REVIEW

File: packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-amount.test.ts [hunk 1]
```diff
@@ -57,7 +71,13 @@ it('should migrate the Amount component', async () => {
   expect(result).toMatchInlineSnapshot(`
     "const App = () => (
             <>
-              <Amount value={1234} color="positive" />
+              <Amount value={1234} color="feedback.text.positive.intense" />
```
Comment (kamleshchandnani) [inline packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-amount.test.ts:74]: the intent 'positive' is mapped to 'intense' of 'positive' color now?

File: packages/blade/src/components/ActionList/styles/getBaseActionListStyles.ts [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
 
 const getBaseActionListStyles = (props: StyledActionListProps & { theme: Theme }): CSSObject => {
-  const { theme, surfaceLevel = 2, isInBottomSheet } = props;
-
-  const backgroundColor = theme.colors.surface.background[`level${surfaceLevel}`].lowContrast;
+  const { theme, isInBottomSheet } = props;
 
   return {
-    backgroundColor,
+    backgroundColor: 'transparent',
```
Comment (kamleshchandnani) [inline packages/blade/src/components/ActionList/styles/getBaseActionListStyles.ts:15]: what if we skip this css style?

**Review by kamleshchandnani — APPROVED** [review]

**Comment by kamleshchandnani** [comment]
> To be merged after #1980

this is 1980 🤔

File: packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-amount.test.ts (hunk 1 above)
Comment (snitin315) [reply packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-amount.test.ts:74]: Yes, this is same as before.

File: packages/blade/src/components/ActionList/styles/getBaseActionListStyles.ts (hunk 2 above)
Comment (snitin315) [reply packages/blade/src/components/ActionList/styles/getBaseActionListStyles.ts:15]: We can skip, no issues.

**Comment by snitin315** [comment]
>this is 1980 🤔

Sorry, I meant after https://github.com/razorpay/blade/pull/1976

File: packages/blade/codemods/brand-refresh/transformers/migrate-amount.ts [hunk 3]
```diff
@@ -98,7 +99,41 @@ function migrateAmountComponent({ root, j, file }): void {
...
     console.error(
-      red(`⛔️ ${file.path}: Oops! Ran into an issue while updating the Amount component.`),
+      red(
+        `⛔️ ${file.path}: Oops! Ran into an issue while updating the "size" prop in "Amount" component.`,
+      ),
+      `\n${red(error.stack)}\n`,
+    );
+  }
+
+  // <Amount intent=”positive”> -> <Amount intent="feedback.text.positive.intense">
```
[the remark this answers is not in the document]
Comment (snitin315) [reply packages/blade/codemods/brand-refresh/transformers/migrate-amount.ts:109]: remove comment?

**Pull request merged**